roles/config-xfce/tasks/xfce.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Install additional packages for XFCE"
```

Comment:
As these are "group" installs, can't use the generic `package` module, so will add code to only support graphical interfaces on Fedora for now